xgboost-predictor/src/main/java/biz/k11i/xgboost/learner/ObjFunction.java

```diff
@@ -115,8 +140,9 @@ float sigmoid(float x) {
      * </p>
      */
     static class RegLossObjLogistic_Jafama extends RegLossObjLogistic {
-        double sigmoid(double x) {
-            return (1 / (1 + FastMath.exp(-x)));
+        @Override
```

Comment:
@honzasterba is this needed for anything? Just curious because we don't bundle Fastmath in H2O-3

Reply:
no, we are not using this, and as it was it did not even work, I saw a warning and I fixed it, thats all